samples/Project GrandMaster/Assets/StreamingAssets/stockfish-10-win/src/benchmark.cpp

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
As far as I understand, we never got this work, should we remove "stockfish" form the sample then?

Reply:
Removed.